src/components/Presentation/index.tsx

```diff
@@ -17,16 +20,43 @@ const TitleAndSwiperContainer = styled(Box)(({ theme }) => ({
 }));
 
 const StyledButton = styled(Button)(({ theme }) => ({
-  height: '48px',
+  height: '52px',
   textTransform: 'none',
   padding: '9px 26px 9px 26px',
   maxHeight: 'fit-content',
-  [theme.breakpoints.down('sm')]: {
-    height: 'fit-content',
+}));
+
+const DiscordButton = styled(Button)(({ theme }) => ({
+  textTransform: 'none',
+  '& span': {
+    margin: '0'
+  },
+}));
+
+const VideoButton = styled(Button)(({ theme }) => ({
+  '& span': {
+    margin: '0'
+  },
+  '& svg': {
+    fontSize: 'xx-large !important'
   },
 }));
 
+const style = {
```

Comment:
Why do we want to use a style variable like this and not a styled component?

Reply:
better avoid this variable and put the styling object directly to the box